convert.go

```diff
@@ -178,12 +325,14 @@ func Convert(to, from Node) (conv Conversion, err error) {
 		sourceToTargetIndex[i] = targetColumn.columnIndex
 	}
 
-	return &conversion{
+	c := &conversion{
 		targetColumnKinds:   targetColumnKinds,
 		targetToSourceIndex: targetToSourceIndex,
 		sourceToTargetIndex: sourceToTargetIndex,
 		schema:              schema,
-	}, nil
+	}
+	c.convertFunc = makeConvertFunc(c, to)
```

Comment:
This is a bit hacky. How do you use `conversion` anywhere in the convert functions?